content/docs/introduction.md

```diff
@@ -3,7 +3,7 @@ title: Neon documentation
 updatedOn: '2024-11-19T20:27:25.090Z'
 ---
 
-Neon is a serverless Postgres platform designed to help you build reliable and scalable applications faster. We separate compute and storage to offer modern developer features such as autoscaling, branching, point-in-time restore, and more. Get started today with our [generous free plan](https://console.neon.tech).
+Neon is a serverless Postgres platform designed to help you build reliable and scalable applications faster. We separate compute and storage to offer modern developer features such as autoscaling, branching, point-in-time restore, and more. Get started today with our <LinkPreview href="https://console.neon.tech" title="PostgreSQL" preview="PostgreSQL is an open source object-relational database system that uses the SQL language and include scalability features">generous free plan</LinkPreview>.
```

Comment:
```suggestion
Neon is a serverless Postgres platform designed to help you build reliable and scalable applications faster. We separate compute and storage to offer modern developer features such as autoscaling, branching, point-in-time restore, and more. Get started today with our [generous free plan](https://console.neon.tech).
```

Reply:
Hi @danieltprice thank you your review! How do you suggest to customize previews titles and texts for external links? 

We discussed this task with Neon team and suggested 2 options: custom component `<LinkPreview>` with props for titles and text for preview, or a separate json file to keep all the data there. So the team chose the first option and we went along with it. 

Do you think a custom json file would be better?

Reply:
@danieltprice may I push our updates for Glossary links right in this PR as well?